### DISCUSSION TITLE
MuMu X 模拟器能用了

### DISCUSSION THREAD

**Discussion by MistEO** [MaaAssistantArknights/MaaAssistantArknights#3389] [General]
我自己试了下，似乎比 蓝叠国际版 还要好用。可能可以该文档了（

**Comment by HS7097** [comment]
> 我自己试了下，似乎比 蓝叠国际版 还要好用。可能可以该文档了（

蓝叠Pie  64bir改成Vulkan/OpenGL一样流畅，而且mumux不能多开，ADB端口共用一个，这点好像还是没变
